prow/cmd/mkbuild-cluster/main.go

```diff
@@ -215,14 +254,22 @@ func describeCluster(o options) (*describe, error) {
 
 // do will get creds for the specified cluster and add them to the stdin secret
 func do(o options) error {
+	var ctx string
```

Comment:
`ctx` usually means a `context.Context`, I'd recommend naming this something else (it'd be like naming a local variable `self` in python)

Reply:
Updated.